Gruntfile.coffee

```diff
@@ -87,13 +87,14 @@ module.exports = (grunt) ->
 
 	)
 
-	@registerTask(
-		"init"
-		"Only needed when the repo is first cloned"
-		[
-			"modernizr"
-		]
-	)
+# Remove modernizr temp
+#@registerTask(
+#		"init"
+#		"Only needed when the repo is first cloned"
+#		[
+#			"modernizr"
```

Comment:
I agree, keep the `grunt init` in package.json and just have an empty task for now